project_tests.sh

```diff
@@ -2,8 +2,9 @@
 set -e
 
 source venv/bin/activate
+export PYTHONOPTIMIZE=
```

Comment:
it was a dangerous option, we put it in to allow mixed host/container local usage as
- host would create `.pyc` files
- container would create `.pyo` files
and despite the host source mounted inside the container, they wouldn't interact or clash in file ownership. I don't think `project_tests.sh` is used much locally so it shouldn't be a problem.